### ISSUE TITLE
Where's my degree sign?

### ISSUE THREAD

**Issue by SebiderSushi** [klausw/hackerskeyboard#613]
In 5-row compact layout I can't find the degree sign.

**Comment by TPS** [comment]
Yeah, in 5-row compact, many symbols from full 5-row are missing.  It does seem that they should be added as just more long-press options (maybe for <kbd>Fn</kbd>?) or just different 1s when <kbd>Shift ⇧</kbd>ed.

**Comment by rogersride** [comment]
Hold down the zero key. the popup shows the degree symbol, third choice to
the left. Hope this helps.

**Comment by SebiderSushi** [comment]
At least for me this only works for 5-Row layout, not for  5-Row compact layout...

**Comment by SebiderSushi** [comment]
Anything new on this topic?

**Comment by SebiderSushi** [comment]
bump

**Comment by SebiderSushi** [comment]
Is there an easy way to modify a keyboard layout?
